tests/test_create.py

```diff
@@ -1405,7 +1405,7 @@ def test_shortcut_absent_does_not_barf_on_uninstall(
     request.addfinalizer(lambda: rmtree(shortcut_file.parent, ignore_errors=True))
 
     # including --no-shortcuts should not get shortcuts installed
-    with tmp_env("console_shortcut", "--no-shortcuts", prefix=prefix):
+    with tmp_env("main::console_shortcut", "--no-shortcuts", prefix=prefix):
```

Comment:
This might add channels to the solver when  `conda-forge` is the default channel. Can we instead use a conda-forge shortcut? e.g. https://anaconda.org/conda-forge/miniforge_console_shortcut

Reply:
See https://github.com/conda/conda/pull/13162/commits/d6748bb3920533320ef2a276f5c4c06f7eb79a86